### PR TITLE
random_forest_regression + test cases added

### PR DESCRIPTION
**Pull request by shreyaspj20**
Partial #1

### CODE REVIEW

File: datascientist/model/regression/skl/ensemble_model/random_forest_regression.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import numpy as np
+
+
+def _random_forest_regression(*, train, test, x_predict=None, metrics, n_estimators=100, max_depth=None,
+                              min_samples_split=2,
+                              min_samples_leaf=1, min_weight_fraction_leaf=0.0, max_features='auto',
+                              max_leaf_nodes=None,
+                              min_impurity_decrease=0.0, min_impurity_split=None, bootstrap=True, oob_score=False,
+                              n_jobs=None, random_state=76,
+                              verbose=0, warm_start=False):
```
Comment (sidhu1012) [inline datascientist/model/regression/skl/ensemble_model/random_forest_regression.py:15]: This is not the right way since function is missing some arguments.

https://github.com/Dacience/DataScientist/blob/5f1371303dcbbb6cbccf9b640b0f2e15f15e1818/datascientist/model/regression/skl/linear_model/baysianridge.py#L9

File: datascientist/model/regression/skl/ensemble_model/random_forest_regression.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    """For for info visit :
+        https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html
+    """
+    model = RandomForestRegressor(n_estimators=n_estimators, max_depth=max_depth, min_samples_split=min_samples_split,
+                                  min_samples_leaf=min_samples_leaf, min_weight_fraction_leaf=min_weight_fraction_leaf,
+                                  random_state=random_state)
+    model.fit(train[0], train[1])
+    model_name = 'RandomForest'
```
Comment (sidhu1012) [inline datascientist/model/regression/skl/ensemble_model/random_forest_regression.py:23]: `RandomForestRegressor`

**Review by sidhu1012 — CHANGES_REQUESTED** [review]

File: datascientist/model/regression/skl/ensemble_model/random_forest_regression.py (hunk 2 above)
Comment (shreyaspj20) [reply datascientist/model/regression/skl/ensemble_model/random_forest_regression.py]: changes done

**Comment by sidhu1012** [comment]
LGTM!

**Comment by shreyaspj20** [comment]
Can I now work on gradient boosting models like xgb, lgbm for regression??

**Comment by shreyaspj20** [comment]
Can I continue working on gradient boosting models under ensemble models folder??

**Comment by sidhu1012** [comment]
> Can I continue working on gradient boosting models under ensemble models folder??

Yes keep going!
